drivers/watchdog/wdt_wwdgt_gd32.c

```diff
@@ -0,0 +1,199 @@
+/*
+ * Copyright (c) 2021 TOKITA Hiroshi
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT gd_gd32_wwdgt
+
+#include <zephyr/drivers/watchdog.h>
+#include <soc.h>
+
+#define WWDGT_NODE DT_NODELABEL(wwdgt)
+
+#define WWDGT_PRESCALER_EXPONENT_MAX    (3U)
+#define WWDGT_COUNTER_MIN               (0x40U)
+#define WWDGT_COUNTER_MAX               (0x7fU)
+#define WWDGT_INTERNAL_DIVIDER          (4096ULL)
+
+/* mutable driver data */
+struct gd32_wwdgt_data {
+	/* counter update value*/
+	uint8_t counter;
+	/* user defined callback */
+	wdt_callback_t callback;
+};
+
+static void gd32_wwdgt_irq_config(const struct device *dev);
+
+static inline struct gd32_wwdgt_data *get_dev_data(const struct device *dev)
+{
+	return dev->data;
+}
```

Comment:
This function is not used and should be removed.

Reply:
Sorry for my late response.
I removed it.